CHANGELOG.md

```diff
@@ -0,0 +1,46 @@
+# Changelog
+
+## [Unreleased]
+
+## v0.2.0 - 2025-04-21
+
+### Breaking Changes
+
+- We now use [traitlets](https://traitlets.readthedocs.io/en/stable/) to handle configuration and command line arguments. Part of [PR #22](https://github.com/2i2c-org/jupyterhub-home-nfs/pull/22). This is a breaking change in how Helm values are passed in:
```

Comment:
Do we need to explain _why_ this change was made in the changelog, or elsewhere and add a link to it?

Reply:
Looks like we don't have an issue for this. Added some explanation in the changelog entry itself.